src/medCore/gui/database/medDatabaseView.cpp

```diff
@@ -559,13 +559,33 @@ void medDatabaseView::onEditRequested(void)
         QList<QVariant> attributes = item->attributes();
         QList<QVariant> values = item->values();
         QList<QString> labels;
-        
+
+        // Users are not allowed to change ThumbnailPath attribute.
+        if (attributes.count() == values.count())
```

Comment:
i finish my task and i look but as far as i remember, there were some possibilities that this happen